### ISSUE TITLE
Add a text label for the "Request to speak" button

### ISSUE THREAD

**Issue by younesaassila** [benawad/dogehouse#363]
I feel like a text label should be added beneath the "Request to speak" button as it may not be obvious to a new user what the button does and would improve accessibility.
